serde_codegen_internals/src/attr.rs

```diff
@@ -746,6 +785,64 @@ fn get_ser_and_de<T, F>(cx: &Ctxt,
     Ok((ser_item.get(), de_item.get()))
 }
 
+type TypeConversions<T> = (Option<T>, Option<T>, Option<T>, Option<T>);
+
+fn get_from_into_as(cx: &Ctxt,
+                    attr_name: &str,
+                    items: &[syn::NestedMetaItem])
+                    -> Result<TypeConversions<syn::Ty>, ()> {
```

Comment:
Get rid of the type parameter on TypeConversions, it is only ever used with syn::Ty.